cmd/webhook/app/options/options.go

```diff
@@ -50,4 +57,6 @@ func (o *Options) AddFlags(flags *pflag.FlagSet) {
 	flags.StringVar(&o.TLSMinVersion, "tls-min-version", defaultTLSMinVersion, "Minimum TLS version supported. Possible values: 1.0, 1.1, 1.2, 1.3.")
 	flags.Float32Var(&o.KubeAPIQPS, "kube-api-qps", 40.0, "QPS to use while talking with karmada-apiserver. Doesn't cover events and node heartbeat apis which rate limiting is controlled by a different set of flags.")
 	flags.IntVar(&o.KubeAPIBurst, "kube-api-burst", 60, "Burst to use while talking with karmada-apiserver. Doesn't cover events and node heartbeat apis which rate limiting is controlled by a different set of flags.")
+	flags.StringVar(&o.MetricsBindAddress, "metrics-bind-address", "0", "The TCP address that the controller should bind to for serving prometheus metrics(e.g. 127.0.0.1:8088, :8088)")
+	flags.StringVar(&o.HealthProbeBindAddress, "health-probe-bind-address", "0", "The TCP address that the controller should bind to for serving health probes(e.g. 127.0.0.1:8080, :8080)")
```

Comment:
metrics and health should have separate address,and should enabled by default.I suggest metrics :8080 health :8000

Reply:
done